l10n_it_ddt/views/stock_picking_package_preparation.xml

```diff
@@ -30,6 +30,10 @@
                            domain="[('type_tax_use','=','sale'),('company_id','=',parent.company_id)]"/>
                 </xpath>
 
+                <field name="date_done" position="attributes">
+                    <attribute name="readonly">0</attribute>
```

Comment:
Come mai permetti di modificare la data di spedizione anche quando il DDT è completato?

Reply:
Perchè può succedere che io prepari il DDT adesso con data di spedizione oggi e poi se il corriere passa domani, vorrei poter cambiare la data a domani per mettere quella di effettiva spedizione

Reply:
Credo che il campo debba riprodurre lo stesso comportamento del `date_done` del picking, e quello non può essere modificato dopo il completamento.
Forse serve un altro campo da poter presentare nella stampa?

Reply:
@SimoRubi : il picking è un documento interno ed è corretto che ci sia un campo date_done che indica quando è stato completato, il DDT è un documento che viene dato al cliente e la data da esporre è "data spedizione" su cui si fa fede per la fatturazione, garanzie prodotto etc.
Non credo che sul DDT sia rilevante la data di completamento del picking
quindi:
su picking date_done
su DDT delivery_date (eventualmente per coerenza di termini) che prende il valore di date_done, ma che può essere modificata a mano, perchè ad esempio oggi stampo i DDT che darò domani al corriere

Reply:
@SimoRubi non ricordo perchè ho marcato questo commento come risolto. Per te è risolto?

Reply:
Visto il caso d'uso, ha senso quindi per me è risolto.
Facevo il parallelo con `stock.picking.date_done` che però non regge perché questo `date_done` è la data in cui è stata fatta la spedizione, che l'utente potrebbe voler cambiare come dice @mrcast.